src/robusta/core/playbooks/prometheus_enrichment_utils.py

```diff
@@ -338,15 +393,15 @@ def create_resource_enrichment(
             values_format=ChartValuesFormat.CPUUsage,
         ),
         (ResourceChartResourceType.Memory, ResourceChartItemType.Pod): ChartOptions(
-            query='sum(container_memory_working_set_bytes{pod=~"$pod", container!="", image!=""}) by (pod, job)',
+            query='sum(container_memory_working_set_bytes{namespace="$namespace", pod=~"$pod", container!="", image!=""}) by (pod, job)',
             values_format=ChartValuesFormat.Bytes,
         ),
         (ResourceChartResourceType.Memory, ResourceChartItemType.Node): ChartOptions(
-            query='instance:node_memory_utilisation:ratio{job="node-exporter", instance=~"$node_internal_ip:[0-9]+"} != 0',
+            query='instance:node_memory_utilisation:ratio{namespace="$namespace", job="node-exporter", instance=~"$node_internal_ip:[0-9]+"} != 0',
```

Comment:
I don't think we need to filter node query with a namespace
Node doesn't have a namespace, it can have pods from multiple namespaces.
Also, when calling the Node type, the labels doesn't have a namespace label